products/secretmanager/terraform.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Secret: !ruby/object:Overrides::Terraform::ResourceOverride
```

Comment:
I think you will need to add custom `import_formats` due to the URL containing `secret_id` instead of `name`. If you look at https://github.com/terraform-providers/terraform-provider-google-beta/pull/1669/files#diff-cc818fd7ea3159b3e18bf79d36939d33R283

I think this will cause imports to fail.

I would also add handwritten import tests because we don't generate them for beta resources